src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
it's enough to implement this interface in Figure